src/utils/shared.ts

```diff
@@ -54,16 +49,18 @@ export const getChainId = (rawDocument: WrappedOrSignedOpenAttestationDocument):
 
   const processChainId = (document: OpenAttestationDocument): number | undefined => {
     if (document.network) {
-      // Check for current bc, "ETH", and chainId, if need cater for other bc and network, update this accordingly. (V2)
-      if (document.network.chain === "ETH" && document.network.chainId) {
+      // Check for current blockchain, "ETH" or "MATIC", and chainId, if need cater for other blockchain and network, update this accordingly.
+      if (AvailableBlockChains.includes(document.network.chain) && document.network.chainId) {
```

Comment:
add test for `MATIC`? do check for a negative scenario too

Reply:
ok